src/server/rendering/fresh_tags.tsx

```diff
@@ -180,5 +187,14 @@ export function renderFreshTags(
     renderState.headVNodes.splice(0, 0, node);
   }
 
+  for (const link of cssLinks) {
+    const node = h("link", {
+      rel: "stylesheet",
+      media: link.media,
+      href: link.url,
+    });
+    renderState.headVNodes.splice(0, 0, node);
+  }
+
```

Comment:
This is all very similar to the `styleTags` stuff. I wonder if there's room for generalization in another PR.